data/authors/Alexandra Harris.toml

```diff
@@ -0,0 +1,11 @@
+name = "Alexandra Harris"
+firstName = "Alexandra"
+lastName = "Harris"
+title = "Newsroom Developer"
+email = "aharris@spotlightpa.org"
+twitter = "harris_alex_m"
+headshot = "2024/11/01km-ace6-mjd2-938y.jpeg"
+reporter = false
+role = "editorial"
+active = true
+content = "Alexandra Harris is the Newsroom Developer at Spotlight PA. Before joining the team, she served as a Newsroom Engineer with Hearst Newspapers, where she focused on full-stack development.\n\nBefore her promotion to Newsroom Engineer, Alexandra worked as a Graphics Reporter. She also held roles as a Data Editor and Fellow on a multi-award-winning series for the Pulitzer Center on Crisis Reporting. She graduated from the University of Florida with technical specializations and experience at The Independent Florida Alligator, WUFT News and the podcast \"Why Don’t We Know.\"\n\nShe is excited to bring her technical expertise to advance Spotlight PA’s mission of public-service journalism."
```

Comment:
The format of this is [TOML](https://toml.io/en/), so you can also do 

```toml
content = """
Alexandra Harris is the Newsroom Developer at Spotlight PA. Before joining the team, she served as a Newsroom Engineer with Hearst Newspapers, where she focused on full-stack development.

Before her promotion to Newsroom Engineer, Alexandra worked as a Graphics Reporter. She also held roles as a Data Editor and Fellow on a multi-award-winning series for the Pulitzer Center on Crisis Reporting. She graduated from the University of Florida with technical specializations and experience at The Independent Florida Alligator, WUFT News and the podcast "Why Don’t We Know."

She is excited to bring her technical expertise to advance Spotlight PA’s mission of public-service journalism.
"""
```

But it's fine as is.